functionalTest/test/specs/example.e2e.js

```diff
@@ -19,53 +19,264 @@ describe("Covid-19 Dashboard", () => {
     dashboard.open();
     expect(dashboard.dailyCases).toBeClickable();
   });
 
-  it("totalCases button should be clickable", () => {
+  xit("totalCases button should be clickable", () => {
     dashboard.open();
     expect(dashboard.totalCases).toBeClickable();
   });
 
-  it("dailyDeaths button should be clickable", () => {
+  xit("dailyDeaths button should be clickable", () => {
     dashboard.open();
     expect(dashboard.dailyDeaths).toBeClickable();
   });
 
-  it("totalDeaths button should be clickable", () => {
+  xit("totalDeaths button should be clickable", () => {
     dashboard.open();
     expect(dashboard.totalDeaths).toBeClickable();
   });
 
-  it("percentageCases button should be clickable", () => {
+  xit("percentageCases button should be clickable", () => {
     dashboard.open();
     expect(dashboard.percentageCases).toBeClickable();
   });
+
+  // Health Boards
+  xit("health board title text and total boards should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    expect(dashboard.healthBoardTitleText).toBeDisplayed();
+    expect(dashboard.healthBoardSubheadingCount).toBeDisplayed();
+  });
+
+  xit("health board total cases text and total cases count should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    dashboard.cases.click();
+    expect(dashboard.healthBoardTotalCasesText).toBeDisplayed();
+    expect(dashboard.healthBoardTotalCasesCount).toBeDisplayed();
+  });
+
+  xit("health board total deaths text and total deaths count should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    dashboard.deaths.click();
+    expect(dashboard.healthBoardTotalDeathsText).toBeDisplayed();
+    expect(dashboard.healthBoardTotalDeathsCount).toBeDisplayed();
+  });
+
+  xit("health board names should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    expect(dashboard.healthBoard01Name).toBeDisplayed();
+    expect(dashboard.healthBoard02Name).toBeDisplayed();
+    expect(dashboard.healthBoard03Name).toBeDisplayed();
+    expect(dashboard.healthBoard04Name).toBeDisplayed();
+    expect(dashboard.healthBoard05Name).toBeDisplayed();
+    expect(dashboard.healthBoard06Name).toBeDisplayed();
+    expect(dashboard.healthBoard07Name).toBeDisplayed();
+    expect(dashboard.healthBoard08Name).toBeDisplayed();
+    expect(dashboard.healthBoard09Name).toBeDisplayed();
+    expect(dashboard.healthBoard10Name).toBeDisplayed();
+    expect(dashboard.healthBoard11Name).toBeDisplayed();
+    expect(dashboard.healthBoard12Name).toBeDisplayed();
+    expect(dashboard.healthBoard13Name).toBeDisplayed();
+    expect(dashboard.healthBoard14Name).toBeDisplayed();
+  });
+
+  xit("health board cases should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    dashboard.cases.click();
+    expect(dashboard.healthBoard01Count).toHaveTextContaining("1995");
```

Comment:
These tests depend on specific dataset inputs or they will fail. That's fine, but we need to make sure that setting the dataset input is part of the test setup or this test will always be very brittle.

Reply:
Noted.